addons/redis-managed/Chart.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+annotations:
+  category: Database
+  licenses: Apache-2.0
+  images: |
+    - name: os-shell
+      image: docker.io/bitnami/os-shell:11-debian-11-r92
+    - name: redis-exporter
+      image: docker.io/bitnami/redis-exporter:1.55.0-debian-11-r3
+    - name: redis-sentinel
+      image: docker.io/bitnami/redis-sentinel:7.2.3-debian-11-r2
+    - name: redis
+      image: docker.io/bitnami/redis:7.2.3-debian-11-r2
+apiVersion: v2
+appVersion: 7.2.3
+dependencies:
+- name: common
+  repository: oci://registry-1.docker.io/bitnamicharts
+  tags:
+  - bitnami-common
+  version: 2.x.x
+description: Redis(R) is an open source, advanced key-value store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets and sorted sets.
+home: https://bitnami.com
+icon: https://bitnami.com/assets/stacks/redis/img/redis-stack-220x234.png
+keywords:
+- redis
+- keyvalue
+- database
+maintainers:
+- name: VMware, Inc.
+  url: https://github.com/bitnami/charts
+name: redis
```

Comment:
yeah good catch I'll change that

Reply:
Since the script is a bit different I'm gonna merge this for now to test and write out the integration, but will follow up to account for this later if that sounds okay